### ISSUE TITLE
Accept-Language Negotiation

### ISSUE THREAD

**Issue by alfaproject** [zfcampus/zf-content-negotiation#13]
http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4
Should this header be supported as a content negotiation option?

I'm in a situation where I have to return different content based on the language that the user chooses in our application, and therefore I was contemplating using a ?lang=en query parameter for it, but then it stroke me that I shouldn't reinvent the wheel. There already exists a http header for that, and browsers are very much aware of it.

The problem at the moment is that I can't directly access this header from a resource, so I will either inject it somewhere in the process, or better yet, add some kind of content negotiation to it, but I don't want to waste my time doing a pull request if it doesn't make sense.

So, basically, I'm requesting feedback on it.
